### ISSUE TITLE
When I change view engine to swig , how can I config cache to 'memory' ?

### ISSUE THREAD

**Issue by perzy** [balderdashy/sails#3031]
Can I config  'cache':'memory'  in sails.config.views file?

**Comment by mahdaen** [comment]
Yep. You can set any Swig configs by set the `config/views.j` like this:

```js
/**
 * Swig Template Rendering Engine.
 */

engine: {
    /* Template File Extension */
    ext : 'swig',

    /* Function to handle render request */
    fn  : function (path, data, cb) {
        /* Binding Configurations */
        data.config = data.sails.config;

        /* Swig Renderer */
        var swig = require('swig');

        /* Disable Caching on Development */
        if ( data.config.environment === 'development' ) {
            swig.setDefaults({ cache : false })
        }

        /* Render Templates */
        return swig.renderFile(path, data, cb);
    }
}
```

**Comment by tjwebb** [comment]
@jerrywuz is this resolved?

**Comment by sailsbot** [comment]
Thanks for posting, @perzy.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
